internal/servers/controller/handler.go

```diff
@@ -74,6 +76,29 @@ func (c *Controller) apiHandler(props HandlerProperties) (http.Handler, error) {
 	return eventsHandler, err
 }
 
+func (c *Controller) GetHealthHandler(lcfg *listenerutil.ListenerConfig) (http.Handler, error) {
```

Comment:
can you add a godoc?

Reply:
Addressed in [8140f0f](https://github.com/hashicorp/boundary/pull/1882/commits/8140f0ffd35b36bdce2ffdc2eafca9070f98615e)